Competition2017/src/competition/subsystems/drive/commands/ResistMovementCommand.java

```diff
@@ -0,0 +1,47 @@
+package competition.subsystems.drive.commands;
+
+import com.google.inject.Inject;
+
+import competition.operator_interface.OperatorInterface;
+import competition.subsystems.drive.DriveSubsystem;
+import competition.subsystems.pose.PoseSubsystem;
+import xbot.common.math.PIDFactory;
+import xbot.common.math.PIDManager;
+import xbot.common.properties.XPropertyManager;
+
+public class ResistMovementCommand extends BaseDriveCommand {
+    PIDManager leftPid;
+    PIDManager rightPid;
```

Comment:
Fixed; good call! It's worth noting that Java's default accessibility is "package protected," meaning that it's accessible to other classes of the same package and inaccessible elsewhere. "package protected" is almost always not what you want.